recipes/pylogbeat/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "pylogbeat" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 539e994e467190273edbada1359465094f0fc9d875d314811f0eb393c9c43978
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - pylogbeat
+
+about:
+  home: "https://github.com/eht16/pylogbeat/"
+  license: Apache Software
```

Needs to be an [SPDX ID](https://spdx.org/licenses/):
```suggestion
  license: Apache-2.0
```